radicle-cli/src/commands/patch.rs

```diff
@@ -24,79 +24,93 @@ pub const HELP: Help = Help {
     usage: r#"
 Usage
 
-    rad patch [<option>...]
+    rad patch
+    rad patch open [<option>...]
+    rad patch update <id> [<option>...]
 
-Create options
+Create/Update options
 
-    -u, --update [<id>]        Update an existing patch (default: no)
+        --[no-]confirm         Don't ask for confirmation during clone
         --[no-]sync            Sync patch to seed (default: sync)
         --[no-]push            Push patch head to storage (default: true)
     -m, --message [<string>]   Provide a comment message to the patch or revision (default: prompt)
         --no-message           Leave the patch or revision comment message blank
 
 Options
 
-    -l, --list                 List all patches (default: false)
         --help                 Print help
 "#,
 };
 
-#[derive(Debug)]
-pub enum Update {
-    No,
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum OptPatch {
     Any,
+    None,
```

Comment:
Doesn't really work because then you can get an update with `None` as the patch_id, which is nonsensical. I'd remove this type completely and just have `Update { patch_id: Option<PatchId> }`

Reply:
Its messy because working on the code required some surgery.  This is an update on the original code which does the same thing.

Both `update` and `create` use a single method whose operations relies on this argument.  Refactoring this meaningfully appeared outside of the issue's scope.